lib/MusicBrainz/Server/Form/User/EditProfile.pm

```diff
@@ -7,9 +7,15 @@ use MusicBrainz::Server::Translation qw( l ln );
 use MusicBrainz::Server::Validation qw( is_valid_url );
 
 extends 'MusicBrainz::Server::Form';
+with 'MusicBrainz::Server::Form::Role::ValidateUsername';
 
 has '+name' => ( default => 'profile' );
 
+has_field 'username' => (
```

Comment:
It's not, specifically, right now, other than by not having a username field in the template on the non-Admin page. Are you thinking we need to implement an extra check in case someone tries to change anyway it by post request or something to that effect?